site/components/src/main/java/uk/nhs/digital/common/components/FeedListComponent.java

```diff
@@ -180,5 +188,21 @@ public void doBeforeRender(HstRequest request,
         request.setAttribute("secondaryButtonText", secondaryButtonText);
         request.setAttribute("secondaryButtonDestination", secondaryButtonDestination);
     }
+
+    @Override
+    public FeedListComponentInfo getComponentParametersInfo(HstRequest request) {
+        FeedListComponentInfo ret = super.getComponentParametersInfo(request);
+        return ret;
+    }
+
+    @Override
+    public Filter createQueryFilter(HstRequest request, HstQuery hstQuery) throws FilterException {
+        return super.createQueryFilter(request, hstQuery);
+    }
+
+    @Override
+    public void applyAndFilters(HstQuery hstQuery, List<BaseFilter> filters) throws FilterException {
+        super.applyAndFilters(hstQuery, filters);
+    }
```

Comment:
Do we need to create above methods here?
getComponentParametersInfo
createQueryFilter
applyAndFilters

Reply:
Yes, as the super methods have protected access modifiers and thus are not accessible outside of their class or sub-class